tests/bsim/bluetooth/mesh/src/test_suspend.c

```diff
@@ -290,6 +408,131 @@ static void test_rx_suspend_disable_resume(void)
 	PASS();
 }
 
+static void test_tx_gatt_suspend_resume(void)
+{
+	bt_mesh_test_cfg_set(NULL, WAIT_TIME);
+	bt_mesh_device_setup(&prov, &comp);
+	ASSERT_OK_MSG(bt_mesh_prov_enable(BT_MESH_PROV_GATT), "Failed to enable GATT provisioner");
+	/* Let the tester observe pb gatt advertisements before provisioning. The node should
+	 * advertise pb gatt service with 100 msec (ext adv) or 1000msec (legacy adv) interval.
+	 */
+	k_sleep(K_MSEC(1800));
+
+	LOG_INF("Provisioning device under test");
+	bt_mesh_provision(test_net_key, 0, 0, 0, tx_cfg.addr, tx_cfg.dev_key);
+
+	/* Let the tester observe proxy advertisements */
+	k_sleep(K_MSEC(6500));
+
+	/* Send a mesh message to notify tester that device is suspended */
+	struct bt_mesh_adv *adv = bt_mesh_adv_create(BT_MESH_ADV_DATA, BT_MESH_ADV_TAG_LOCAL,
+			BT_MESH_TRANSMIT(0, 20), K_NO_WAIT);
+	net_buf_simple_add_mem(&adv->b, "1", sizeof("1"));
```

Comment:
It is better to use meaningful messages rather than 0 or 1. Smth. like `suspend` and `resume` to have self-descriptive source code that doesn't require comments.

Reply:
Changed to enum.